spec/system/catalog_show_spec.rb

```diff
@@ -142,7 +142,7 @@
     let(:details_fields) do
       ['printer', 'binder', 'former-owner', 'notes_display', 'binding_note_display',
        'provenance_display', 'references_url_display', 'other_format_display',
-       'other_title_display', 'recap_notes_display']
```

Comment:
Why are we removing `recap_notes_display`?

Reply:
it is not in the updated fixture